setup.py

```diff
@@ -15,7 +15,8 @@
       author_email = 'Christoph.Freundl@ifm.com',
       license = 'MIT',
       packages = ['o3d3xx', 'o3d3xx.rpc', 'o3d3xx.pcic'],
-      install_requires = ['future'],
+      scripts = ['scripts/o3d3xx-updater.py'],
+      install_requires = ['future','requests'],
```

Comment:
I think I would like to avoid that the requests library is becoming a mandatory prerequisite for the whole o3d3xx-python package.

Reply:
You are right. I am quite unsure if the updater fits very well into this lib or is it worth to create an own project.